mix.exs

```diff
@@ -32,7 +32,7 @@ defmodule TelemetryMetricsAppsignal.MixProject do
       {:hammox, "~> 0.2", only: :test},
       {:jason, "~> 1.1", optional: true},
       {:telemetry, "~> 0.4 or ~> 1.0"},
-      {:telemetry_metrics, "~> 0.4"}
+      {:telemetry_metrics, "~> 0.6"}
```

Comment:
Is this necessary? It's not like we don't support version 0.4 anymore - that will still work, right?

Reply:
You're right. The new features are optional so we can leave it up to the user to upgrade. I'll remove this commit.